### DISCUSSION TITLE
Textures too stretchy?

### DISCUSSION THREAD

**Discussion by gyarisuta** [MenacingMecha/godot-psx-style-demo#23] [Q&A]
Is this normal at all? i dont feel like this is meant to work this way.
[Image](https://cdn.discordapp.com/attachments/866907496323874846/1232803943059423415/Godot_v4.2.1-stable_win64_5bQ94B13Wb.gif?ex=662ac9a4&is=66297824&hm=d6e40d4724da48366e17cc958c863f1d48f51cf4e927f77b7a232826bfcabb62&)
Here is the shadermaterial config 

![Godot_v4 2 1-stable_win64_wvK40IZWyv](https://github.com/MenacingMecha/godot-psx-style-demo/assets/72901838/e5411ca1-76de-418d-8d0e-262ae1164c0b)

**Comment by MenacingMecha** [comment]
Image has expired - please don't link to Discord media, it's ephemeral.

Taking a swing with no context, are there not enough edge loops in the mesh?  It's expected behavior that textures warp towards vertices with the turning of the camera.
